### DISCUSSION TITLE
Activate chromaprint from command line

### DISCUSSION THREAD

**Discussion by das-s** [beetbox/beets#4426] [Q&A] [answered]
Hi, is there a way to activate the chroma plugin in the command line explicitly? 

Idea is having the plugin loaded, but not enabled:
```
chroma:
  auto: no
```
And then (e.g. in certain directories with bad tags) enable the fingerprinting during the call to `beet import` via a command flag. Is that currently possible?

A search in the documentation, Discourse and Github has not shown any results for me.

**Comment by arsaboo** [comment]
`beet fingerprint` should generate fingerprints for items in your library ([docs](https://beets.readthedocs.io/en/stable/plugins/chroma.html#usage)).

**Reply by das-s** [reply]
Thanks, but that only works after the import, right? I'd like to activate chromaprint only for certain imports with bad metadata /during the import process/, before they get into my library.

**Reply by arsaboo** [reply]
Don't think that is possible. Either you run it for all the tracks at the time of import or run it selectively post-import.

**Comment by sampsyo** [comment, marked as answer]
One option is to use an overlay configuration. That is, you could put into `add_chromaprint.yaml`:

```yaml
chroma:
    auto: true
```

or similar, and then do `beet -c add_chromaprint.yaml import ...` to enable fingerprinting.

**Reply by das-s** [reply]
Thanks, that works. I did not realize that `-c` overlays the existing config (instead of replacing it).
